loopchain/blockchain/exception.py

```diff
@@ -41,6 +41,12 @@ class AddUnconfirmedBlock(Exception):
     pass
 
 
+class AlreadyAddedUnconfirmedBlock(RuntimeWarning):
```

Comment:
Is there any reason to use `RuntimeWarning` instread of `RuntimeError` or `Exception`?

Reply:
I would like to classify clearly.

Reply:
But it it used and works like normal exceptions.
Warning does not interrupt flow of functions. But in this case it interrupts as normal exceptions do.

You should use `warnings` module instead of `raise`.
https://python.flowdas.com/library/warnings.html

Reply:
I changed RuntimeWarning to Exception.